Slims/slims.py

```diff
@@ -0,0 +1,183 @@
+import os
+import re
+
+from slims.slims import Slims
+from slims.criteria import is_one_of, equals, conjunction, not_equals
+from slims.content import Status
+
+
+
+class slims:
+    url = os.environ.get('SLIMS_URL')
+    user = os.environ.get('SLIMS_USER')
+    password = os.environ.get('SLIMS_PASSWORD')
+
+
+instance = 'wgs-somatic_query'
+url = slims.url
+user = slims.user
+password = slims.password
+slims = Slims(instance, url, user, password)
+
+
+
+class SlimsSample:
+    def __init__(self, sample_name, run_tag=''):
+        self.sample_name = sample_name
+        self.run_tag = run_tag
+
+        self._dna = None
+        self._fastq = None
+        self._bioinformatics = None
+
+    @property
+    def dna(self):
+        if not self._dna:
+            records = slims.fetch('Content', conjunction()
+                                  .add(equals('cntn_id', self.sample_name))
+                                  .add(equals('cntn_fk_contentType', 6)))
+
+            if len(records) > 1:
+                raise Exception('More than 1 DNA somehow.')
+
+            if records:
+                #print(records)
+                self._dna = records[0]
+
+        return self._dna
+
+    @property
+    def fastq(self):
+        if not self.run_tag:
+            raise Exception('Can not fetch fastq without a set run tag.')
+        print(self.run_tag)
+        print(self.sample_name)
+        if not self._fastq:
+            records = slims.fetch('Content', conjunction()
+                                  .add(equals('cntn_id', self.sample_name))
+                                  .add(equals('cntn_fk_contentType', 22))
+                                  .add(equals('cntn_cstm_runTag', self.run_tag)))
+            print(records) 
+            if len(records) > 1:
+                raise Exception('More than 1 fastq somehow.')
+
+            if records:
+                self._fastq = records[0]
+
+
+        # if sample_name has fastqs from additional sequencing runs - fetch those fastq objects
+        more_fastqs = slims.fetch('Content', conjunction()
```

Comment:
I think this functionality should either be it's own separate function or at the very least a function belonging to the class.